### DISCUSSION TITLE
Welcome to teps Discussions!

### DISCUSSION THREAD

**Discussion by josecelano** [torrust/teps#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cgbosse** [comment]
Hi everyone, 
I am Constantin a colleague of Jose's
I am the only non tech person. I help with various things non technical, where I can be of help, such as reviewing documentation, for spelling and grammar, to helping with structuring and planning overviews for the project.
